packages/w3up-client/src/client.js

```diff
@@ -54,6 +55,31 @@ export class Client extends Base {
     this.coupon = new CouponAPI(agentData, options)
   }
 
+  /**
+   * Verify and return Abilities.
+   *
+   * Given a list of strings representing capability names (Abilities),
+   * verify that all the strings are valid Abilities and return Abilities[].
+   *
+   * Abilities[] is still just a list of strings, but this helps us play
+   * nice with Typescript.
+   *
+   * @param {string[]} abilities
+   * @returns {import('@web3-storage/access').Abilities[]}
+   */
+  static abilities(abilities) {
```

Comment:
I'd export this as a function instead: `asAbilities(abilities: string[]): Abilities[]`